pandas/core/frame.py

```diff
@@ -2301,6 +2301,7 @@ def to_stata(
         time_stamp: datetime.datetime | None = None,
         data_label: str | None = None,
         variable_labels: dict[Hashable, str] | None = None,
+        value_labels: dict[str, dict[float | int, str]] | None = None,
```

Comment:
Since these are not keyword only, I don't think this can be inserted here since it could produce a crash ins otherwise correct code. Would be great to move most of these to be keyword only, but that is a different issue.

Reply:
I still think this might need to move to the end of the list. Ideally you would make it keyword only, e.g.,

```
,
*,
value_labels: dict[Hashable, dict[float | int, str]] | None = None,
```

Also note that it should be `dict[Hashable, ...` not `str`